### PR TITLE
Refactor mount06 using new LTP API

### PR DESCRIPTION
**Pull request by acerv**
Signed-off-by: Andrea Cervesato <andrea.cervesato@suse.com>

### CODE REVIEW

File: testcases/kernel/syscalls/mount/mount06.c [hunk 1]
```diff
@@ -1,34 +1,19 @@
...
+ * [Description]
+ *
+ * Test for feature MS_MOVE of mount, which moves an existing mount point to
+ * a new location.
  */
 
-#include <errno.h>
+#include "tst_test.h"
+#include <stdio.h>
 #include <sys/mount.h>
```
Comment (richiejp) [inline testcases/kernel/syscalls/mount/mount06.c:16]: Aren't these headers already included?

File: testcases/kernel/syscalls/mount/mount06.c [hunk 2]
```diff
@@ -38,126 +23,77 @@
...
 {
-	if (mount_flag && tst_umount(path_name) != 0)
-		tst_resm(TWARN | TERRNO, "umount(2) %s failed", path_name);
+	SAFE_MOUNT(tst_device->dev, mntpoint_src, tst_device->fs_type, 0, NULL);
+	SAFE_MOUNT(mntpoint_src, mntpoint_dst, tst_device->fs_type, MS_MOVE, NULL);
 
-	if (device)
-		tst_release_device(device);
+	TST_EXP_EXPR(
+		!tst_is_mounted(mntpoint_src) && tst_is_mounted(mntpoint_dst),
```
Comment (richiejp) [inline testcases/kernel/syscalls/mount/mount06.c:70]: Just a suggestion: This isn't a very strong check. It should probably try to access the files as in the previous test.

**Review by richiejp — CHANGES_REQUESTED** [review]
Need to add signed-off-by on the actual commit

**Comment by pevik** [comment]
> Need to add signed-off-by on the actual commit

@acerv `git config --global alias.c 'commit -s'` would add help you not to forget to add `Signed-off-by:`. It expects to have correct email setup.
NOTE: You can easily switch working and private email over `$GIT_AUTHOR_EMAIL` and `$GIT_COMMITTER_EMAIL` [environment variables](https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables).

**Pull request closed without merging**